test/e2e/storage/external/external.go

```diff
@@ -255,6 +255,10 @@ func (d *driverDefinition) SkipUnsupportedTest(pattern testpatterns.TestPattern)
 		if d.SnapshotClass.FromName || d.SnapshotClass.FromFile != "" || d.SnapshotClass.FromExistingClassName != "" {
 			supported = true
 		}
+	case testpatterns.PreprovisionedCreatedSnapshot:
```

Comment:
Since this check is exactly the same, can we combine the two cases into one line?